its/src/test/java/com/sonar/maven/it/suite/LinksTest.java

```diff
@@ -35,19 +34,13 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.groups.Tuple.tuple;
 
-public class LinksTest extends AbstractMavenTest {
-
-  @Before
-  @After
-  public void cleanProjectLinksTable() {
-    orchestrator.getDatabase().truncate("project_links");
-  }
```

Comment:
It looks like since we call `AbstractMavenTest::resetData` after each test, the links table should be cleared from the database but maybe we could twisted the logic by relying on the actual implementation.
```java
 public void cleanProjectLinksTable() {
    var database = orchestrator.getDatabase();
    ((DefaultDatabase) database).start(); // This should be idempotent
    database.truncate("project_links");
  }
````